pushapkscript/test/integration/test_integration_script.py

```diff
@@ -99,51 +105,62 @@ def _copy_single_file_to_test_temp_dir(self, task_id, origin_file_name, destinat
             os.makedirs(target_dir)
             shutil.copy(original_path, target_path)
 
-    @unittest.mock.patch('mozapkpublisher.push_apk.PushAPK')
-    def test_main_downloads_verifies_signature_and_gives_the_right_config_to_mozapkpublisher(self, PushAPK):
+    @unittest.mock.patch('pushapkscript.googleplay.push_apk')
+    def test_main_downloads_verifies_signature_and_gives_the_right_config_to_mozapkpublisher(self, push_apk):
         task_generator = TaskGenerator()
         task_generator.generate_file(self.config_generator.work_dir)
 
         self._copy_all_apks_to_test_temp_dir(task_generator)
         main(config_path=self.config_generator.generate())
 
-        PushAPK.assert_called_with(config={
-            'credentials': '/dummy/path/to/certificate.p12',
-            '*args': sorted([
-                '{}/work/cot/{}/public/build/target.apk'.format(self.test_temp_dir, task_generator.arm_task_id),
-                '{}/work/cot/{}/public/build/target.apk'.format(self.test_temp_dir, task_generator.x86_task_id),
-            ]),
-            'commit': False,
-            'credentials': '/dummy/path/to/certificate.p12',
-            'no_gp_string_update': True,
-            'service_account': 'dummy-service-account@iam.gserviceaccount.com',
-            'track': 'alpha',
-        })
-
-    @unittest.mock.patch('mozapkpublisher.push_apk.PushAPK')
-    def test_main_allows_rollout_percentage(self, PushAPK):
+        push_apk.assert_called_with(
+            [
+                MockFile(
+                    '{}/work/cot/{}/public/build/target.apk'.format(self.test_temp_dir, task_generator.arm_task_id)),
+                MockFile(
+                    '{}/work/cot/{}/public/build/target.apk'.format(self.test_temp_dir, task_generator.x86_task_id)),
+            ],
+            'dummy-service-account@iam.gserviceaccount.com',
+            MockFile('/dummy/path/to/certificate.p12'),
+            'alpha',
+            None,
+            ANY,
```

Comment:
Until we're on Python 3.7+, I can't make `NoGooglePlayStrings`/`FileGooglePlayStrings`/etc `@dataclasses`. And, until I do that, I can't just `assert_called_with` where the parameters are the expected classes.

So, in the meantime, I've `ANY`'d here, and added a section at the bottom of the test to pull out the `[5]`th parameter and assert that it's `NoGooglePlayStrings`. It's not ideal, but not baaaad